graphdatascience/graph/graph_object.py

```diff
@@ -41,24 +49,59 @@ def _graph_info(self, yields: List[str] = []) -> "Series[Any]":
         return info.squeeze()  # type: ignore
 
     def database(self) -> str:
+        """
+
+        Returns:
+            the name of the database the graph is stored in
+        """
         return self._graph_info(["database"])  # type: ignore
 
     def configuration(self) -> "Series[Any]":
+        """
+
```

Comment:
Nitpick: There's a lot of inconsistency with the blank lines here. Would be nice to fix that